TesseractOCR/G8Tesseract.mm

```diff
@@ -644,6 +645,49 @@ - (NSString *)recognizedHOCRForPageNumber:(int)pageNumber {
     return nil;
 }
 
+- (NSData *)recognizedPDFForImages:(NSArray*)images {
```

Comment:
I'm not sure I totally understood your motivation for adding Tesseract's PDF rendering support until now, but if it's truly only to generate PDFs of images, do we really need to use Tesseract's PDF renderer? Couldn't the user just use one of the existing iOS methods for rendering a PDF with embedded images? For example, it looks like you could do it directly with Apple's iOS APIs: https://developer.apple.com/library/ios/documentation/2DDrawing/Conceptual/DrawingPrintingiOS/GeneratingPDF/GeneratingPDF.html

What does Tesseract's PDF renderer provide or do over other PDF rendering methods?